.travis.yml

```diff
@@ -41,6 +41,8 @@ before_install:
         ;;
     esac
 
+    pip install pylint
```

Comment:
@jart, you know Travis better than I do—would there be any advantages to making this its own yaml item? or, more generally, breaking this block into three: one to install bazel, one to install TensorFlow, and one to install pylint?